fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -51,6 +51,16 @@ public ConcreteAvailability(
     }
 
     @Override
+    public SimplifiedIntervalList getAvailableIntervals() {
+        return null;
```

Comment:
This doesn't look right... Isn't this just asking for NPE elsewhere at a random place?